lib/babel-config.js

```diff
@@ -25,68 +24,12 @@ function validate(conf) {
 	return conf;
 }
 
-function lazy(initFn) {
-	let initialized = false;
-	let value;
-
-	return () => {
-		if (!initialized) {
-			initialized = true;
-			value = initFn();
-		}
-
-		return value;
+function makeTransformTestFiles(powerAssert) {
+	return babel => {
+		return require('@ava/babel-preset-transform-test-files')(babel, {powerAssert});
 	};
 }
 
-const defaultPresets = lazy(() => {
```

Comment:
It seemed unnecessary yesterday but I think I figured it out. This config is used even if all transforms have been cached. So we should keep the laziness in place.